src/Turbo/composer.json

```diff
@@ -67,10 +51,20 @@
         "symfony/messenger": "^5.2",
         "symfony/panther": "^1.0",
         "symfony/phpunit-bridge": "^5.2.1",
+        "symfony/property-access": "^5.2",
```

Comment:
this one is optional IIUC

Reply:
More or less, in practice most (all?) transports will need it.